even-more-fish-plugin/src/main/java/com/oheers/fish/placeholders/PlaceholderReceiver.java

```diff
@@ -4,143 +4,137 @@
+                                Priority.SPECIFIC_PATTERN,
+                                this::handleCompetitionPlaceFish
+                        ),
+                        new HandlerDefinition(
+                                id -> id.startsWith("competition_place_player_"),
+                                Priority.SPECIFIC_PATTERN,
+                                this::handleCompetitionPlacePlayer
+                        ),
+
+                        // Total stats placeholders
+                        new HandlerDefinition(
+                                id -> id.startsWith("total_money_earned_"),
+                                Priority.GENERAL_PATTERN,
+                                this::handleTotalMoneyEarned
+                        ),
+                        new HandlerDefinition(
+                                id -> id.startsWith("total_fish_sold_"),
+                                Priority.GENERAL_PATTERN,
+                                this::handleTotalFishSold
+                        ),
+
+                        // Exact match placeholders
+                        new HandlerDefinition(
+                                id -> id.equalsIgnoreCase("competition_time_left"),
+                                Priority.EXACT_MATCH,
+                                this::handleCompetitionTimeLeft
+                        ),
+                        new HandlerDefinition(
+                                id -> id.equalsIgnoreCase("competition_active"),
+                                Priority.EXACT_MATCH,
+                                this::handleCompetitionActive
+                        ),
+                        new HandlerDefinition(
+                                id -> id.equalsIgnoreCase("custom_fishing_boolean"),
+                                Priority.EXACT_MATCH,
+                                this::handleCustomFishingBoolean
+                        ),
+                        new HandlerDefinition(
+                                id -> id.equalsIgnoreCase("custom_fishing_status"),
+                                Priority.EXACT_MATCH,
+                                this::handleCustomFishingStatus
+                        ),
+                        new HandlerDefinition(
+                                id -> id.equalsIgnoreCase("competition_type"),
+                                Priority.EXACT_MATCH,
+                                this::handleCompetitionType
+                        ),
+                        new HandlerDefinition(
+                                id -> id.equalsIgnoreCase("competition_type_format"),
+                                Priority.EXACT_MATCH,
+                                this::handleCompetitionTypeFormat
+                        )
+                ).stream()
+                .sorted(Comparator.comparingInt(def -> def.priority().ordinal()))
+                .collect(Collectors.toList());
```

Comment:
You can just directly call `.toList()` instead of `.collect(Collectors.toList())